pages/common/views/components/filter-summary.jsx

```diff
@@ -2,13 +2,15 @@ import React from 'react';
 
 const FilterSummary = ({
   all,
-  filtered
+  filtered,
+  filteredLabel = `Showing ${filtered.length} of ${all.length} results`,
+  allShowingLabel = `All ${all.length} results`
 }) => (
   <h2 className="filter-summary">
     {
       filtered.length !== all.length
-        ? `Showing ${filtered.length} of ${all.length} results`
-        : `All ${all.length} results`
+        ? filteredLabel
+        : allShowingLabel
     }
   </h2>
 );
```

Comment:
This one might end up being tricky to pass props into as strings because template literals are resolved at definition time, so it won't be possible to pass a template literal to be resolved in context.

That's future-us's problem though.